### PR TITLE
Todo list testing 

### PR DESCRIPTION
**Pull request by Bria222**
-  Created a test file  following functions:

1. a function for editing the task description.
2. a function for updating an item's 'completed' status.
3. The "Clear all completed" function.

- Used the mock storage object that we created in Part 1 of this project to mock the LocalStorage updates.

### CODE REVIEW

File: src/editTask.test.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+  const checkBoxValue = document.querySelector('.text-input');
+  const textValue = document.querySelector('.text-value');
+
+  test('Should edit task description', () => {
+    taskObj.addNewTask(textValue);
+    document.body.innerHTML = '  <div class="todo-list"></div> ';
+    displayTask();
+    expect(taskObj.tasks).toHaveLength(1);
+  });
```
Comment (uwadonat) [inline src/editTask.test.js:37]: - [ ] Let's add test for  "Clear all completed" function.

**Review by uwadonat — CHANGES_REQUESTED** [review]
Hi **Team,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Your commit and pull request messages are very descriptive ✔️
- No linter errors ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [x] Kindly add instructions to test your project in Readme file. It will make your project more professional.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/editTask.test.js (hunk 1 above)
Comment (Bria222) [reply src/editTask.test.js:37]: @uwadonat okay we will add the steps to run the tests on the readme file.

Comment (Bria222) [reply src/editTask.test.js:37]: Screenshot (65)

We have included the steps and the test results on the readme file
thanks for your response we have worked on implementing the change

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi team 👋🏻, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️

- [x] As requested by the previous reviewer, Kindly include tests for clearing all the completed tasks

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Bria222** [comment]
@Cathella  but I have done that I have that function inside the edit task.js file

**Comment by Bria222** [comment]
Screenshot (67)_LI

**Review by ShahierNashaat — APPROVED** [review]
Hi @Bria222 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**